src/App.js

```diff
@@ -1,26 +1,31 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+
+  const addMovie = (newMovie) => {
+    if (!newMovie || movies.some(movie => movie.imdbID === newMovie.imdbID)) {
```

Comment:
```suggestion
    if (movies.some(movie => movie.imdbID === newMovie.imdbID)) {
```
You always pass movie to this function in your code, so there's no need to check if it is present